docs/modules/demos/pages/argo-cd-git-ops.adoc

```diff
@@ -0,0 +1,317 @@
+* Secrets: How to safely store and deploy credentials, access keys from a Git repository?
+* Interaction with Stackable products (e.g. Airflow and DAGs via gitsync)
+
+ArgoCD and other deployed products and dependencies are illustrated in the following diagram: 
+
+image::argo-cd-git-ops/architecture-overview.drawio.svg[]
+
+Install this demo on an existing Kubernetes cluster:
+
+TODO: We need a pointer here to the last paragraph (forking repo) if people want to "interact" and not just install the demo (install command requires extra parameters)
+
+[source,console]
+----
+$ stackablectl demo install argo-cd-git-ops --namespace argo-cd
+----
+
+WARNING: This demo should not be run alongside other demos.
+
+NOTE: ArgoCD will be deployed in the `argo-cd` namespace by `stackablectl`. ArgoCD itself will create other namespaces for the deployed products.
+
+[#system-requirements]
+== System requirements
+
+To run this demo, your system needs at least:
+
+* 15 {k8s-cpu}[cpu units] (core/hyperthread)
+* 15 GiB memory
+* 5 GiB disk storage
+
+== Overview
+
+This demo consists of multiple parts:
+
+* Bootstrapping via `stackablectl`:
+** Install {argo-cd}[Argo CD] via `stackablectl`
+* Deploy components via ArgoCD:
+** Install a {sealed-secrets}[Sealed Secrets] controller to handle sensitive data like credentials or secret keys
+** Install all Stackable operators using an `ApplicationSet`
+** Install requirements like Minio and Postgres as `Application`
+** Deploy Stackable Airflow manifests into their respective `Projects`
+* Inspect ArgoCD UI
+* Inspect Airflow and start DAGs
+** Checkout Airflow UI and synced DAGs (the DAGs are synced via gitsync, not ArgoCD)
+** Start a DAG and inspect the logs written to S3 / Minio by the Kubernetes executor
+* Optional: Inspect Minio UI for logs
+
+== Sealed Secrets
+
+When managing all resources and configs via Git, deploying sensitive properties like certificates or credentials via Git becomes a problem.
+
+There are multiple solutions like Hashicorp or Bitwarden, which heavily depend on the infrastructure already available.
+
+For the sake of this demo, {sealed-secrets}[Bitnami's Sealed Secrets] are utilized. 
+Sensitive data is encrypted as a `SealedSecret` before commiting to the Git repository, synced via ArgoCD and decrypted by the Sealed Secrets controller into a standard Kubernetes `Secret`.
+
+This way, everything will be stored and managed in Git.
+
+== ArgoCD UI
+
+ArgoCD will be the first product that is deployed in this Demo. Once the pods are ready, you can port-forward the argocd-server in order to access the web UI.
+
+[source,console]
+----
+kubectl --namespace argo-cd port-forward service/argocd-server 8443:https
+----
+
+In your browser, go to `https://localhost:8443` and login with username `admin` and password `adminadmin`.
+
+NOTE: There will be an initial warning from the Browser, that this site is insecure due to self-signed certificates. This can be ignored in this case.
```

Comment:
```suggestion
NOTE: There will be an initial warning from the browser, stating that the site is insecure due to self-signed certificates.
This can be ignored in this case.
```